src/RAWcookedBundle/Resources/views/Default/index.html.twig

```diff
@@ -106,40 +106,40 @@
-<p>This way the contributing bodies and individuals are sure their files can be processed by the software.</p>
-<p>The price is 1&nbsp;000&nbsp;&euro; for basic support (bug fixes) and one non-free flavour, then 500&nbsp;&euro; for each following non-free flavour (additional flavours can be added later if you are not sure of how many flavours you need now, at the same price).</p>
+<p>MediaArea.net has achieved implementation of RAWcooked’s basic module with thanks to AV Preservation reto.ch, who has guaranteed software development with funding of 10,000€.</p>
+<p>The archive community or interested industry users are invited to <a href="{{ path('ma_contact') }}">buy</a> additional flavours required by their institution - such as components, bit depths, endianess, packing, etc.  Required flavours will be fully tested by MediaArea.net to exact specification before being implemented into the software.</p>
+<p>This model allows contributors the assurance that their files will work seamlessly with the software from the outset.</p>
+<p>The price is 1000€ for basic support including bug fixes and one additional flavour, then 500 € for each following flavour.  Additional flavours can be added at a later if you are not sure of how many flavours you need immediately.</p>
 <h2>Status</h2>
-<p>RAWcooked is currently under development. <a href="/download/snapshots/binary/rawcooked/">Development snapshots</a> are available (<a href="https://github.com/MediaArea/RAWcooked">corresponding code</a>), not to be used in production yet. As in January, some DPX flavors are supported, a sequence of DPX is detected, encode is delegated to FFmpeg and decode is native (from another MediaArea project), <code>rawcooked yourdpx.dpx</code> provides the FFmpeg command to run for encode, <code>rawcooked yourdpx.mkv</code> reverts bit-by-bit.<br>
-Priority is given to formats and flavors from our sponsors. You can test with this <a href="https://github.com/MediaArea/RAWcooked-RegressionTestingFiles/blob/master/Features/AV_Package/818_DCDM_P3_IA_FIC_000918/818_OV/818_OV_0086945.dpx?raw=true">file</a> (or a <a href="https://github.com/MediaArea/RAWcooked-RegressionTestingFiles/tree/master/Features/AV_Package/818_DCDM_P3_IA_FIC_000918/818_OV">sequence of files</a>).</p>
+<p>RAWcooked had its first official release October 2018, with support for a limited number of DPX flavours. To use RAWcooked simply type  <code>rawcooked /RAWsequenceDPX/</code> which begins the encoding using FFmpeg to create an MKV encoded file in the same folder, leaving the original DPX sequence in tact.  You can also use this command <code>rawcooked yourdpx.dpx</code> using the first DPX file in a sequence within your folder. To return the MKV back to its RAW source format use <code>rawcooked yourdpx.mkv</code> which uses RAWcooked software to unpack your RAW sequence bit-by-bit, including all original metadata and sidecar files.<br>

```

Comment:
"in tact" seems weird, wanted?

Reply:
It should read 'intact', sorry! Meaning, not damaged or impaired in any way, complete.

Reply:
Double spaces at `type <code>rawcooked` and `tact. You`.